common/djangoapps/student/management/tests/test_bulk_user_activate.py

```diff
@@ -0,0 +1,67 @@
+"""Tests for the bulk_user_activate command."""
+
+import ddt
+from django.core.management import call_command
+from django.core.management.base import CommandError
+
+from student.tests.factories import UserFactory
+from student.models import User
+from xmodule.modulestore.tests.django_utils import SharedModuleStoreTestCase
+
+from student.models import get_users_by_email
+
+
+@ddt.ddt
+class BulkUserActivateTests(SharedModuleStoreTestCase):
+    """Tests for the bulk_user_activate command."""
+
+    def setUp(self):
+        super(BulkUserActivateTests, self).setUp()
+        self.users = UserFactory.create_batch(5)
```

Comment:
Use `User.objects.create(username='user1', email='user1@domain1.com', is_active=False)` to create some inactive users, and also a few active ones to check that it just does it on inactive users